apps/site/public/types/modules/graph/0.3/schema/aggregation-operation.json

```diff
@@ -1,87 +1,125 @@
+                      "operator": {
+                        "$ref": "#/$defs/FilterOperatorRequiringValue"
+                      },
+                      "value": {
+                      }
+                    },
+                    "required": [
+                      "field",
+                      "operator",
+                      "value"
+                    ],
+                    "additionalProperties": false
                   },
-                  "value": {
-                    "type": "string"
+                  {
+                    "type": "object",
+                    "properties": {
+                      "field": {
+                        "$ref": "#/$defs/Field"
+                      },
+                      "operator": {
+                        "$ref": "#/$defs/FilterOperatorWithoutValue"
+                      }
+                    },
+                    "required": [
+                      "field",
+                      "operator"
+                    ],
+                    "additionalProperties": false
                   }
-                },
-                "type": "object"
+                ]
+              }
+            },
+            "operator": {
+              "$ref": "#/$defs/MultiFilterOperatorType"
+            }
+          },
+          "required": [
+            "filters",
+            "operator"
+          ],
+          "additionalProperties": false
+        },
+        "multiSort": {
+          "type": "array",
+          "items": {
+            "type": "object",
+            "properties": {
+              "field": {
+                "$ref": "#/$defs/Field"
               },
-              {
-                "properties": {
-                  "field": {
-                    "type": "string"
+              "desc": {
+                "oneOf": [
+                  {
+                    "type": "boolean"
                   },
-                  "operator": {
-                    "$ref": "#/$defs/FilterOperatorWithoutValue"
+                  {
+                    "type": "null"
                   }
-                },
-                "type": "object"
+                ]
               }
-            ]
-          },
-          "type": "array"
-        },
-        "operator": {
-          "$ref": "#/$defs/MultiFilterOperatorType"
+            },
+            "required": [
+              "field"
+            ],
+            "additionalProperties": false
+          }
         }
       },
-      "type": "object"
+      "additionalProperties": false
     },
-    "multiSort": {
+    "graphResolveDepths": {
+      "$ref": "https://blockprotocol.org/types/modules/graph/0.3/schema/partial-graph-resolve-depths"
+    }
+  },
+  "required": [
+    "operation"
+  ],
+  "additionalProperties": false,
+  "$defs": {
+    "Field": {
+      "type": "array",
       "items": {
-        "properties": {
-          "desc": {
-            "type": "boolean"
-          },
-          "field": {
+        "anyOf": [
```

Comment:
Similarly `oneOf` would be more correct here, `string & number` is unsatisfiable

Reply:
Fixed in [Apply suggestions from Code Review](https://github.com/blockprotocol/blockprotocol/pull/1034/commits/2e5bd9dc964ed1713eb1ec846335a89862f8e4a5)